test/css/component_specific_selectors_test.rb

```diff
@@ -79,6 +79,14 @@ class ComponentSpecificSelectorsTest < Minitest::Test
       ".Button--iconOnly.Button--small",
       ".Button--iconOnly.Button--large"
     ],
+    Primer::Beta::ButtonGroup => [
+      ".BtnGroup + .BtnGroup",
+      ".BtnGroup+.BtnGroup",
+      ".BtnGroup + .btn",
+      ".BtnGroup+.btn",
```

Comment:
@mxriverlynn Not sure how to make the `CSS Coverage` check pass with these `+` sibling selectors. 

[It fails](https://github.com/primer/view_components/actions/runs/3536603923/jobs/5935785853#step:7:15) with:

```
.BtnGroup+.BtnGroup
.BtnGroup+.btn
```

even when added to `IGNORED_SELECTORS`? 🤔

Reply:
the ignored selectors are all treated as regex, so you would need to escape it like `/\+/` ... i think that should capture anything with a `+` in the selector

unfortunately, we don't really have a way to parse through all the selector rules for anything more than just a very basic, single selector. there may be some way to do it, but i don't know how at the moment

Reply:
@mxriverlynn Thanks. 🙇 Adding `/\+/` to global resolved it and yeah, might be good to have that anyways for future sibling selectors. 👍